interface-spec.md

```diff
@@ -66,15 +68,15 @@
 
 **Properties:**
 
-- `Node .graph` the named graph, which is an IRI, a BlankNode or NodeVariable.
+- `Node .why` the provenence, which may be a named node represenitng a document on the web which has this graph, or a BlankNode or NodeVariable. Also extednable for the results of infreence.
```

Comment:
Discussed in the 2015-11-27 meeting: `.graph` is the more generic meaning; it is not necessarily for provenance.

However, individual implementations are free to extend objects with other properties, such as `.why`. In general, we have to be careful about the handling of such extra properties, when serializing, cloning, etc. Other examples include line number, DOM element, …